gobblin-modules/gobblin-azkaban/src/main/java/org/apache/gobblin/service/modules/orchestration/AzkabanClient.java

```diff
@@ -130,10 +138,12 @@ private void initializeClient() throws AzkabanClientException {
 
   private void initializeSessionManager() throws AzkabanClientException {
     if (sessionManager == null) {
-      this.sessionManager = new AzkabanSessionManager(this.httpClient,
```

Comment:
If we follow the code here I don't think this is needed, it's just constructing the object so it should never throw.